### DISCUSSION TITLE
Suggestions for mini workshop!

### DISCUSSION THREAD

**Discussion by starlightknown** [starlightknown/OpenX#1] [Ideas]
Hey folks, I need few tips/ideas for a mini workshop
So here's what I planned - intro
Some theory about the topic
Hands on experience in the workshop itself plus it'll be available for them to practice later. Topic is open source and basic things about GitHub. For hands on I've created a smol site where they can put up a PR to be added as a contributor so that their first PR is merged plus it'll be there for them to remember for long time. So if you were attending a workshop what would you expect and any suggestions would be appreciated. Thank you.

**Comment by vinzvinci** [comment]
• Yes basic first, introduce the platform, how to get started.
• Benefits of the platform and how this platform will help them in their career.
• Show some demo for beginner only.
• Promote Student Developer Pack in education.github.com

**Comment by felixfaisal** [comment]
I think @vinzvinci has covered everything, I'd like to add two things to it 
- If it's for beginners, Then you need to lure them in first that is tell them **why** open source before teaching them **how** to open source.
- Contributions to open source is not limited to code, it could be designing, auditing, documentation or even a tutorial. All of this counts as a contribution. It's one thing most people do not talk about while addressing beginners.

**Comment by starlightknown** [comment]
Thank you @felixfaisal & @vinzvinci ! will surely include these points! 💯

**Comment by eddiejaoude** [comment]
Looks great @starlightknown and great points by everyone 🎉 

Just a little note from me, I would suggest having a clear beginning, say what you will cover and have a glossary, as people won't know some abbreviations are, for example `repo` is short for `repository. Then at the end, have a conclusion with a summary and maybe some actions for them to do, like customise their GitHub profile

**Comment by jainaayush01** [comment]
Nice points by everyone.
- Also, Give them some examples of Open Source Softwares they have been using.
- Benefits like if they have a nice profile on GitHub or have done some nice Open Source work, they can also be reached out by recruiters, it will help them in getting Freelancing clients, etc.

**Comment by WebShivam** [comment]
I think almost all points are covered by everyone here. I'd just like you to suggest only one thing:
-  Introducing the agenda of workshop at the very beginning of stream, So they can know what to expect from the workshop which will excite them to stick with it till the end

**Comment by suubh** [comment]
In the end you can tell them to do a course by **GitHub** .You can provide them this [link.](https://lab.github.com/githubtraining/introduction-to-github)
It will help them to move further after your workshop .

**Comment by starlightknown** [comment]
thank you all
